unit_test/inputs/hertzian_contact.json

```diff
@@ -11,6 +11,6 @@
     "final_time"             : {"value": 1e-4, "unit": "s"},
     "timestep"               : {"value": 1e-8,  "unit": "s"},
     "timestep_safety_factor" : {"value": 0.8},
-    "output_frequency"       : {"value": 50},
+    "output_frequency"       : {"value": 1000},
```

Comment:
I don't think there is a way to completely disable output for this as a "unit" test. I'll leave it to you to decide what's the best approach here.